About.html

```diff
@@ -0,0 +1,146 @@
+              <a href="#" class="menu-link">ِِJoin</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link">ِِSponser</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link">ِِNews</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link active-border">ِِGDC Campaign</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <section class="two">
+      <div class="transparent2">
+        <h4 class="text-hello">"Hello! Sharing World"</h4>
+        <h1 class="textabout">Game Developers Conference 2022</h1>
+      </div>
+    </section>
+    <section class="fourth-section">
+      <div class="fourth-content">
+        <h6 class="item-text">
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+          culpa qui officia deserunt mollit anim id est laborum."
+        </h6>
+      </div>
+      <p class="fourth-text">
+        please contact us per Email for any furthur questions about GDC Global
+        Summit 2022!.
+      </p>
+      <a href="fourth-text">gdccontact@gmail.com</a>
+    </section>
+    <section class="fifth-section">
+      <h2 class="text5">Gamer Developers Conference Logo 2022</h2>
+      <div class="line2"></div>
+      <p class="item-text">
+        The logo of the GDC was decided through the logo competition from 8.june
+        to 7.july.
+      </p>
+      <img class="logo5" src="images/GDC.png" alt="logo" />
+    </section>
+    <section class="sixth-section">
+      <h2 class="text5">See the past GDC Global Summits</h2>
+      <div class="line2"></div>
+      <p class="item-text">
+        Take a look at the last 2 GDC global Summits which took place in buenos
+        Aires and in Warsaw
+      </p>
+      <div class="pictures">
+        <div class="picture">
+          <div class="trans2">
+            <h2 class="text6-1">2013</h2>
+            <h2 class="text6">GDC Global summit in Buenos Aires</h2>
+          </div>
+        </div>
+        <div class="picture2">
+          <div class="trans2">
+            <h2 class="text6-1">2011</h2>
+            <h2 class="text6">GDC Global summit in Warsaw</h2>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-content">
+        <img class="logo6" src="images/GDC.png" alt="logo" />
+        <h2 class="footer-text">
+          2022 Gamer Developer Conference San Francisco, some rights reserved.
+        </h2>
+      </div>
+    </footer>
```

Comment:
- Kindly follow the background color and other color guidelines used for the footer of the About page 😀